catalog/transformer/catalog-transformer-xml/src/main/java/ddf/catalog/transformer/xml/adapter/ObjectAdapter.java

```diff
@@ -51,6 +57,14 @@ public static ObjectElement marshalFrom(Attribute attribute) throws CatalogTrans
   public static Attribute unmarshalFrom(ObjectElement element) {
     AttributeImpl attribute = null;
     for (Serializable value : element.getValue()) {
+      try {
```

Comment:
❓ Should we first check that it is a `byte[]`?

Reply:
`ObjectElement.getValue()` does return a list of `byte[]`, but you're right that relying on that implementation would be coupling the code. I'll make a check there first